### ISSUE TITLE
Can import pyfolio in Ipython but not jupyter 

### ISSUE THREAD

**Issue by MichJim** [quantopian/pyfolio#403]
I tried to install pyfolio for the first time today.

My default python version is 2.7, did a $ pip install there

Tried in Python3, did a $ pip install there.  (python3   -m pip install pyfolio - install was completed)

In Jupyter notebook, I can access both Python2 and Python3.

after the $ pip install, I tried 'import pyfolio as py'

in both versions of Python in Jupyter, I got an error message 'ModuleNotFoundError: No module named 'pyfolio''.  (I am not a prgrammer - is there something else I should be doing...).  I have closed everything down, reopened etc

I opened Ipython in a new shell,  tried the 'import pyfolio as py', and seemed to work ok.

My impression is that the package works better in Jupyter so would be best to use it in there?

**Comment by twiecki** [comment]
Are you using anaconda? I highly recommend installing that and trying again.

**Comment by MichJim** [comment]
Thanks.  I had been using miniconda, will install anaconda

**Comment by twiecki** [comment]
hm, miniconda should be fine. There is definitely a problem with the environment / installation. Not quite sure which. does `pip install pyfolio` followed by `python -c "import pyfolio"` work? That points to jupyter.

**Comment by twiecki** [comment]
Ah, I think I know. I bet you're still using your system python somehow (either to install into or to run). miniconda should tell you to add something to your .bashrc file to automatically make you use the newly installed one.

**Comment by MichJim** [comment]
tied the python -c "import pyfolio" but it didnt work

No.2 is prob correct, but not sure how I can resolve it

**Comment by MichJim** [comment]
Thanks for the help - I am not an experienced Python user

**Comment by MichJim** [comment]
(I was resigned to trying to make it all work in Ipython, which seems to work ..)

**Comment by kyangvandy** [comment]
I have a similar issue. I have installed pyfolio using anaconda, and have no problem import it in pycharm. But when I tried to import in jupyter notebook, it returned the following errors. Have you guys figured out how to resolve the issue?
Thanks in advance. 
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-47-1977abd82007> in <module>()
----> 1 import pyfolio as pf

ModuleNotFoundError: No module named 'pyfolio'
